pkg/cmd/pulumi/deployment_settings_config.go

```diff
@@ -248,9 +243,61 @@ func newDeploymentSettingsInitCmd() *cobra.Command {
 		&stack, "stack", "s", "",
 		"The name of the stack to operate on. Defaults to the current stack")
 
+	cmd.PersistentFlags().BoolVarP(
+		&force, "force", "f", false,
+		"Forces content to be generated even if it is already configured")
+
 	return cmd
 }
 
+func initStackDeploymentCmd(d *deploymentSettingsCommandDependencies, gitSSHPrivateKeyPath string) error {
+	d.Deployment = &workspace.ProjectStackDeployment{
+		DeploymentSettings: apitype.DeploymentSettings{},
+	}
+
+	err := configureGit(d, gitSSHPrivateKeyPath)
+	if err != nil {
+		return err
+	}
+
+	err = configureImageRepository(d)
+	if err != nil {
+		return err
+	}
+
+	err = configureAdvancedSettings(d)
+	if err != nil {
+		return err
+	}
+
+	option := promptUserSkippable(
+		d.Yes,
```

Comment:
Hmm I know we often use `--yes` for confirming prompts, but it feels quite strange here to confirm a prompt where the options are "No" and some other options with `--yes`.  Maybe having flags for each of the options and `--skip-openid-configure` might be better?

Reply:
I am using `yes` as a way to automatically confirm every prompt with the default option, I can change that but wanted to avoid the explosion of flags

Reply:
I think it's fine to add more flags if they are useful to the user, which I think they would be here.

Reply:
I believe it would be better to tackle this on a different PR